x-pack/plugins/session_view/public/components/SessionView/index.tsx

```diff
@@ -123,7 +129,31 @@ export const SessionView = ({ sessionEntityId, height }: SessionViewDeps) => {
             onProcessSelected={onProcessSelected}
           />
         </div>
-      )}
+      );
+    }
+    return (
+      <EuiEmptyPrompt
+        iconType="alert"
+        color="danger"
+        title={<h2>Error loading Session View</h2>}
+        body={<p>There was an error loading the Session View.</p>}
+      />
+    );
+  };
+
+  const toggleDetailPanel = () => {
+    setIsDetailMounted(!isDetailMounted);
+    if (!isDetailOpen) setIsDetailOpen(true);
```

Comment:
nitpick: brackets

Reply:
oh... this shouldn't be here. looks like some merge conflicts didn't get resolved properly when i switched from the `session-detail-view` branch

Reply:
just double checked, there are no other conflicts